### PR TITLE
Example to demo differences between the image shader and mesh

### PR DESCRIPTION
**Pull request by hmaarrfk**
Closes: https://github.com/pygfx/pygfx/issues/799

### CODE REVIEW

**Comment by Korijn** [comment]
I wonder if a grid would be useful here to show the 0.5 pixel offset?

**Comment by hmaarrfk** [comment]
I didn't want to be too negative about the grid object, but I currently find it quite jarring.

Moving around in the example i can generate the following displays:

```
$ python examples/feature_demo/line_plot.py
```

image
image
image
image

It makes me really lose track of what the grid is about, and so I was hoping to take a look at it eventually before using it in a real example.

My attempt at providing a visual guide is with a red line at the top.

File: examples/feature_demo/image_scaling.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
+"""
+Displaying an image in a scene
+==============================
+
+Displaying an image can be done one of two different ways based the details
```
Comment (almarklein) [inline examples/feature_demo/image_scaling.py:5]:
```suggestion
Displaying an image can be done in two different ways based the details
```

File: examples/feature_demo/image_scaling.py [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+is the important information. By default, nearest neighbor interpolation
+is used to sample the texture.
+
+A gfx.Mesh with a plane geometry can be used to display the texture with
+spanning from the one edge of the geometry to an other.
+The pixel corners at integer coordinates. This is useful when one wants
+to stretch the provided data to fill a specific area. Be default bilinear
+interpolation is used to sample the texture.
+
+The example provide allows users zoom in and out of the scene to study the
```
Comment (almarklein) [inline examples/feature_demo/image_scaling.py:22]:
```suggestion
The example allows users zoom in and out of the scene to study the
```

**Review by almarklein — COMMENTED** [review]
Just 2 typos. Otherwise looks like a nice addition to the gallery!

**Comment by hmaarrfk** [comment]
> Just 2 typos. Otherwise looks like a nice addition to the gallery!


ps. you can just push in these case to avoid you having to come back to this PR. your choice.

**Review by Korijn — APPROVED** [review]

**Pull request merged**